packager/media/formats/mp4/segmenter.cc

```diff
@@ -216,7 +180,14 @@ Status Segmenter::FinalizeSegment(size_t stream_id,
           {key_frame_info.timestamp, moof_start_offset,
            fragment_buffer_->Size() - moof_start_offset + key_frame_info.size});
```

Comment:
Linking [the now 'outdated' discussion](https://github.com/CaitlinOCallaghan/shaka-packager/pull/1/files/9ebbfd1c6dd055abc41d0af338914c5a3d0ba310#r660997429) to the new change to prevent the thread from disappearing from main view.